### PR TITLE
Fixes #29410 - Use correct event for mounting chart in job details

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

**Comment by adamruzicka** [comment]
[test foreman_remote_execution]

File: app/views/job_invocations/show.html.erb [hunk 1]
```diff
@@ -1,7 +1,9 @@
 <% title @job_invocation.description %>
 <% stylesheet 'foreman_remote_execution/foreman_remote_execution' %>
 <% javascript 'charts', 'foreman_remote_execution/template_invocation' %>
-<%= javascript_include_tag *webpack_asset_paths('foreman_remote_execution', :extension => 'js'), "data-turbolinks-track" => true, 'defer' => 'defer' %>
+<% content_for :javascripts do %>
+  <%= javascript_include_tag *webpack_asset_paths('foreman_remote_execution', :extension => 'js') %>
```
Comment (ezr-ondrej) [inline app/views/job_invocations/show.html.erb:5]: Maybe a `javascript` can be used for `webpack` assets as well? It should then go in the header as well, right?

**Comment by ezr-ondrej** [comment]
Who angried the dog? :( and who can merge this? :thinking:

**Comment by ares** [comment]
@ezr-ondrej hound seems calm, you should now have permissions :wink:

**Comment by ezr-ondrej** [comment]
Thanks @adamruzicka ! :+1:

**Pull request merged**